modules/installation/proc-deploy-rhdh-instance-gke.adoc

```diff
@@ -0,0 +1,252 @@
+      enabled: false
+    appConfig:
+      configMaps:
+        - name: "app-config-rhdh"
+    extraEnvs:
+      secrets:
+        - name: "secrets-rhdh"
+----
+--
+
+. Set up a Google-managed certificate by creating a `ManagedCertificate` object which you will attach to the Ingress.
++
+--
+[source,yaml,subs="attributes+"]
+----
+apiVersion: networking.gke.io/v1
+kind: ManagedCertificate
+metadata:
+  name: <rhdh_certificate_name>
+spec:
+  domains:
+    - <rhdh_domain_name>
+----
+--
+For more information about setting up a Google-managed certificate, see https://cloud.google.com/kubernetes-engine/docs/how-to/managed-certs?hl=en#setting_up_a_google-managed_certificate[Setting up a Google-managed certificate]. 
+
+. Create a `FrontendConfig` object to set a policy for redirecting to HTTPS. You will attach this policy to the Ingress. 
++
+--
+[source,yaml,subs="attributes+"]
+----
+apiVersion: networking.gke.io/v1beta1
+kind: FrontendConfig
+metadata:
+  name: <ingress_security_config>
+spec:
+  sslPolicy: gke-ingress-ssl-policy-https
+  redirectToHttps:
+    enabled: true
+----
+--
+For more information about setting a policy to redirect to HTTPS, see https://cloud.google.com/kubernetes-engine/docs/how-to/ingress-configuration?hl=en#https_redirect[HTTP to HTTPS redirects].
+
+. Create an Ingress resource using the following template, customizing the names as needed:
++
+--
+[source,yaml,subs="attributes+"]
+----
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  # TODO: this the name of your Developer Hub Ingress
+  name: my-rhdh
+  annotations:
+    # If the class annotation is not specified it defaults to "gce".
+    kubernetes.io/ingress.class: "gce"
+    kubernetes.io/ingress.global-static-ip-name: <ADDRESS_NAME>
+    networking.gke.io/managed-certificates: <rhdh_certificate_name>
+    networking.gke.io/v1beta1.FrontendConfig: <ingress_security_config>
+spec:
+  ingressClassName: gce
+  rules:
+    # TODO: Set your application domain name.
+    - host: <rhdh_domain_name>
+      http:
+        paths:
+        - path: /
+          pathType: Prefix
+          backend:
+            service:
+              # TODO: my-rhdh is the name of your Backstage Custom Resource.
+              # Adjust if you changed it!
+              name: backstage-my-rhdh
+              port:
+                name: http-backend
+----
+--
+
+. Wait for the `ManagedCertificate` to be provisioned. This can take a couple of hours.
+
+. Access RHDH with `https://<rhdh_domain_name>`
+
+.Verification
+
+Wait until the DNS name is responsive, indicating that your {product-short} instance is ready for use.
+
+.Additional information
+For more information on setting up {gke-short} using Ingress with TLS, see https://github.com/GoogleCloudPlatform/gke-networking-recipes/tree/main/ingress/single-cluster/ingress-https 
+
+For more information on setting up {gke-short} with LoadBalancer instead of Ingress, see https://github.com/sumiranchugh/rhdh-gke-poc/tree/main 
```

Comment:
This link is missing the link text.

Reply:
@zdrapela What would be an approriate heading to use here to describe the content at this link - the linked GitHub repo does not have any obvious title?

Reply:
Link has been removed